init.sh

```diff
@@ -218,6 +218,7 @@ export DOCKER_TAG=${DOCKER_TAG:-latest}
 export LIGHTHOUSE_DOCKER_TAG=${LIGHTHOUSE_DOCKER_TAG:-latest}
 REGENERATE=${REGENERATE:-0}
 SLEEP_TIME=${SLEEP_TIME:-5}
+MAINTENANCE_MODE=${MAINTENANCE_MODE:-0}
```

Comment:
May it be worth adding some documentation about this in the Readme for catalyst owners?

Reply:
Yes, forgot about that. Thanks